src/extendPrototype.ts

```diff
@@ -1,13 +1,39 @@
 /* eslint-disable @typescript-eslint/ban-types */
-export default function extendProtoype(
-    target: Function,
-    func: Function,
-    functionName: string
+export function extendProtoype(
+  target: Function,
+  func: Function,
+  functionName: string
 ): void {
-    if (!Object.getOwnPropertyNames(target.prototype).includes(functionName)) {
-        Object.defineProperty(target.prototype, functionName, {
-            value: func,
-            writable: true,
-        })
-    }
+  if (!Object.getOwnPropertyNames(target.prototype).includes(functionName)) {
```

Comment:
The hot new way to do this is `Object.hasOwn()` - shorter and safer in some edge cases